### ISSUE TITLE
Unexpected token 'U'

### ISSUE THREAD

**Issue by essimoon** [stackblitz/bolt.new#1517]
### Describe the bug

I'm getting this error opening my project. It still worked earlier on, not sure what this means:
Unexpected token 'U', "Unexpected"... is not valid JSON

### Link to the Bolt URL that caused the error

https://bolt.new/~/sb1-jvhjfi

### Steps to reproduce

Click on the URL and see if the project imports properly.

### Expected behavior

The project imports and opens properly.

### Platform

```
Browser name = Chrome
Full version = 128.0.0.0
Major version = 128
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/128.0.0.0 Safari/537.36 OPR/114.0.0.0
performance.memory = {
  "totalJSHeapSize": 107294506,
  "usedJSHeapSize": 102551898,
  "jsHeapSizeLimit": 4294705152
}
Username = essimoon
Chat ID = 18eadbc8fd3d
```

### Additional context

Error:
```
An unknown error occured while running an action.
```

**Comment by arjunw7** [comment]
Is there a way to reach out to support?
I am middle of a meeting and bolt has stopped working.

**Comment by essimoon** [comment]
> Is there a way to reach out to support? I am middle of a meeting and bolt has stopped working.

oh, so it's a global issue? Thanks for the heads-up. Unfortunately I don't know how to contact them directly

**Comment by essimoon** [comment]
Update: it works again!

**Comment by kc0tlh** [comment]
@essimoon Thanks for the report and apologies for the frustrating experience! The fix is explained in #1471
